data-prepper-expression/src/test/java/org/opensearch/dataprepper/expression/ParserTest.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.expression;
+
+import org.antlr.v4.runtime.TokenStream;
+import org.antlr.v4.runtime.tree.ParseTree;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.opensearch.dataprepper.expression.antlr.DataPrepperExpressionParser;
+import org.opensearch.dataprepper.expression.util.MockTokenStreamHelper;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.Mockito.doAnswer;
+import static org.opensearch.dataprepper.expression.util.ContextMatcherFactory.isParseTree;
+import static org.opensearch.dataprepper.expression.util.TerminalNodeMatcher.isTerminalNode;
+
+@ExtendWith(MockitoExtension.class)
+public class ParserTest {
+    private static final Class<? extends ParseTree> Expression = DataPrepperExpressionParser.ExpressionContext.class;
```

Comment:
The Java convention for `static final` is to use all caps : `EXPRESSION`. These are private, but I still think it would make the code more readable.